src/test/resources/170.315_b1_toc_gold_sample2_v1.xml

```diff
@@ -23,7 +23,7 @@
 	<realmCode code="US"/>
 	<typeId extension="POCD_HD000040" root="2.16.840.1.113883.1.3"/>
 	<!-- US Realm Header ID-->
-	<templateId root="2.16.840.1.113883.10.20.22.1.1" extension="2015-08-01"/>
+	<templateId root="2.16.840.1.113883.10.20.22.1.1" extension="2015-08-01"></templateId>

```

Comment:
Why add a closing tag to this and not the other two?

Reply:
I didn't, at least not intentionally. we can discard this change, it doesn't do anything.